redux/ducks/collections.js

```diff
@@ -0,0 +1,66 @@
+import createReducer from 'type-to-reducer';
+
+import omit from 'lodash/omit';
+
+import api from 'constants/api';
+import { defaultReducer } from 'utils/redux';
+import { makeRequest } from 'utils/request';
+import { getLocalizedCollection, getLocalizedArticles, getArticlesBlocks } from 'utils/getters';
+
+const duck = 'collections';
+
+// constants
+const FETCH_COLLECTION = `${duck}/FETCH_COLLECTION`;
+
+// reducer
+const initialState = {
+  pending: false,
+  error: false,
+  collection: null,
+};
+
+export default createReducer(
+  {
+    [FETCH_COLLECTION]: defaultReducer((state, { payload }) => ({
+      ...state,
+      ...payload,
+    })),
+  },
+  initialState
+);
+
+// selectors
+const getState = state => state.collections;
+const getData = (state, lang) => {
+  const { collection } = getState(state);
+  const localizedArticles = getLocalizedArticles(collection.articles, lang);
+
+  const authors = Object.values(
+    localizedArticles.reduce((acc, { tagsByTopic: { authors: articleAuthors } }) => {
+      articleAuthors.forEach(authorTag => {
+        acc[authorTag.slug] = authorTag;
+      });
+      return acc;
+    }, {})
+  );
+
+  return {
+    collection: getLocalizedCollection(omit(collection, ['articles']), lang),
```

Comment:
why omit?

Reply:
Collection contains articles, each article contain reference to collection. So we are localizing recursively here, that is not necessary. Articles are delivered separatly as blocks.